### DISCUSSION TITLE
在webpack5项目中如何脱离umi独立使用mfsu呢

### DISCUSSION THREAD

**Discussion by shengkan9** [umijs/umi#8166] [Q&A] [answered]
我的项目是基于webpack5版本的，想使用umi4.0的mfsu方案，我在网上查找了些相关文档修改了webpack配置，使用的版本是"@umijs/mfsu": "^4.0.0-beta.9"，但是启动项目时编译会报错如图，请问如何解决
![image](https://user-images.githubusercontent.com/74705339/174567095-f934345c-580d-4623-8d99-67aa05f1b390.png)

**Comment by xiaohuoni** [comment, marked as answer]
https://umijs.org/blog/mfsu-independent-usage 根据文档走走看
